dev/ci/docker_linux/Dockerfile

```diff
@@ -67,10 +68,12 @@ RUN apt-get update && apt-get install -y --no-install-recommends \
   ruby-dev && \
   apt-get clean
 
-ENV JAVA_HOME="/usr/lib/jvm/adoptopenjdk-8-hotspot-amd64"
+ENV JAVA_HOME="/usr/lib/jvm/adoptopenjdk-11-hotspot-amd64"
+
+RUN export CLASSPATH='/usr/share/jaxb/lib'
 
 # Install the Android SDK Dependency.
-ENV ANDROID_SDK_URL="https://dl.google.com/android/repository/sdk-tools-linux-4333796.zip"
+ENV ANDROID_SDK_URL="https://dl.google.com/android/repository/commandlinetools-linux-9477386_latest.zip"
```

Comment:
Is there a helpful comment you can put here on how to update next time?

Reply:
Yeah I can add a note. I think in this case it was just Android updating their packaging and me not being aware of it until this PR. But I will add a note.